concrete_biopython/FheSeq.py

```diff
@@ -485,7 +485,7 @@ def translate(self, table="Standard"):
         return self.__class__(protein_seq, self._fhe_seq_maker)
 
 
-    def complement(self, inplace=False):
+    def _complement(self, inplace=False):
```

Comment:
Why convert to private methods?

Reply:
no good reason I can remeber of, I changed it :)

Reply:
because one of the child class does not have the inplace=False argument

Reply:
How is it connected to being private?

Reply:
that was because the abstract class was calling self._complement(inplace) which did not exist in both child class with an argument, so I used super().complement(inplace) now